operator/templates/node-scripts.yaml

```diff
@@ -1,20 +1,30 @@
+{{ $auth_params := "" }}
+{{ if .Params.AUTHENTICATION_SECRET_NAME }}
```

Comment:
we should also populate the credentials for cqlsh when authentication is enabled
https://docs.datastax.com/en/security/5.1/security/secCqlshCredentials.html
otherwise `cqlsh` queries will fail

Reply:
This should be up to the admin. We don't use `cqlsh` as a utility in the operator. The idea is that admins set up authn/authz to their liking, then provide the operator with a secret containing credentials for the operator internal use of `nodetool`.

Reply:
https://github.com/kudobuilder/operators/blob/7e4eeb595a0f8efbb73e151b26d7c0b7d9133ec9/repository/cassandra/3.11/docs/accessing.md#access-cassandra 
this won't work. So once inside pod, we should configure it accordingly so users don't have to do manual setup to gain access.

Reply:
This will work if users add `--user` and `--password` to their call :D
While I agree, that we could add the credentials of the secret to every `cqlshrc` in the cluster, this would only provide a single user for every call to `cqlsh`. Which is what authn/authz wants to avoid. Users calling `cqlsh` should have their own credentials and different authorization rules based on those credentials.

Reply:
ok yeah, you are right! 
I will mark this as resolved. I would like to see that being provided out of the box but we will provide this feature once we have any request for that and probably with another parameter 👍